resim/metrics/proto/validate_metrics_proto.py

```diff
@@ -829,14 +829,16 @@ def validate_job_metrics(job_metrics: mp.JobMetrics) -> None:
         metric_data_names.add(metric_data.name)
         _validate_metrics_data(metric_data, metrics_data_map)
 
-    # Use a set to check for duplicated names
-    event_names = set()
-    # Validate that all events per-job use the same timestamp type:
-    timestamp_type = job_metrics.events[0].timestamp_type
-    for event in job_metrics.events:
-        _metrics_assert(event.name not in event_names)
-        _metrics_assert(event.timestamp_type == timestamp_type)
-        event_names.add(event.name)
-        _validate_event(event, metrics_map)
+    if job_metrics.events:
+        # Use a set to check for duplicated names
+        event_names = set()
+
+        # Validate that all events per-job use the same timestamp type:
+        timestamp_type = job_metrics.events[0].timestamp_type
+        for event in job_metrics.events:
+            _metrics_assert(event.name not in event_names)
+            _metrics_assert(event.timestamp_type == timestamp_type)
+            event_names.add(event.name)
+            _validate_event(event, metrics_map)
```

Comment:
I think this will cause coverage issues which I'll try to fix.

Reply:
ta